src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -54,6 +54,9 @@ public static class EnvironmentSettingNames
         public const string AzureWebsiteRunFromPackage = "WEBSITE_RUN_FROM_PACKAGE";
         public const string RegionName = "REGION_NAME";
 
+        // handling server side builds
+        public const string ScmBuildPackage = "SCM_BUILD_PACKAGE";
```

Comment:
appsetting name doesnt look right.

Reply:
ah, this one should be SCM_RUN_FROM_PACKAGE